jobs/check-aws-regions.Jenkinsfile

```
@@ -0,0 +1,33 @@
+import org.yaml.snakeyaml.Yaml;
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+}
+
+properties([
+    pipelineTriggers([
+        githubPush(),
+        // once a day
+        pollSCM('H H * * *')
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '100',
+        artifactNumToKeepStr: '100'
```

```suggestion
        numToKeepStr: '20',
        artifactNumToKeepStr: '20'
```

Should be fine to reduce this number so we don't eat up storage for something that's not really super useful